### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by koppor**
Enjoy!

### CODE REVIEW

File: src/main/resources/l10n/JabRef_es.properties [hunk 1]
```diff
@@ -2285,21 +2290,11 @@ Don't\ share=No compartir
...
-Names\ are\ not\ in\ the\ standard\ %0\ format.=Los nombres no tienen el formato estándar
-
-Delete\ the\ selected\ file\ permanently\ from\ disk,\ or\ just\ remove\ the\ file\ from\ the\ entry?\ Pressing\ Delete\ will\ delete\ the\ file\ permanently\ from\ disk.=¿Borrar el archivo seleccionado del disco duro o solamente quitarlo de la entrada? Presionar el botón de eliminara, borrará el archivo del disco duro permanentemente.
-Delete\ '%0'=Borrar archivo '%0'
-Delete\ from\ disk=Borrar del disco duro
-Remove\ from\ entry=Quitar de la entrada
```
Comment (tobiasdiez) [inline src/main/resources/l10n/JabRef_es.properties]: @koppor Why does Crowdin revert the changes of https://github.com/JabRef/jabref/pull/3833 ?

**Comment by lenhard** [comment]
No idea. I was blindly assuming that crowdin does the right thing, especially since the build passed. 

Maybe it conflicts with localizations that are added directly via a PR? Should I just re-add the removed translations on master?

**Comment by koppor** [comment]
After merging such PRS, the JabRef crowidn admins (me, @siedlerchr) habe to
push "stop sync" and "start sync" to trigger an import

Am 13.03.2018 16:53 schrieb "Jörg Lenhard" <notifications@github.com>:

**Comment by Siedlerchr** [comment]
That's why I was waiting with merging the Spanish PR as I was unsure if crowdin gets the changes.
Good to know that in such cases we have to re-trigger the sync
